src/fastbt/utils.py

```diff
@@ -565,23 +565,23 @@ def get_nearest_premium(
     1. nearest premium is calculated on the basis of absolute difference
     """
     if instrument_map is None:
-            return []
+        return None
     
-    if isinstance(instrument_map,dict):  
+    elif isinstance(instrument_map,dict):  
         
-        difference=None
+        diff=None
         latest_symbol=None
         for symbols,ltp in instrument_map.items():
             last_trade_price=int(float(ltp))
```

Comment:
why are you forcing this to be int

Reply:
this test works in my system.Kindly check again if possible

Reply:
this one passes